cdk/lib/bandit.ts

```diff
@@ -103,68 +136,17 @@ export class Bandit extends GuStack {
 			inputPath: '$.Payload',
 		});
 
-		const banditsTable = new Table(this, 'bandits-table', {
-			tableName: `support-bandit-${this.stage}`,
-			removalPolicy: RemovalPolicy.RETAIN,
-			billingMode: BillingMode.PAY_PER_REQUEST,
-			partitionKey: {
-				name: 'testName',
-				type: AttributeType.STRING,
-			},
-			sortKey: {
-				name: 'timestamp',
-				type: AttributeType.STRING,
-			},
-			pointInTimeRecovery: this.stage === 'PROD',
-			timeToLiveAttribute: 'ttlInSeconds',
-		});
 
-		const calculateLambda = new GuLambdaFunction(this, 'calculate-lambda', {
-			app: appName,
-			functionName: `${appName}-calculate-${this.stage}`,
-			runtime: Runtime.NODEJS_20_X,
-			handler: 'calculate-lambda/calculate-lambda.run',
-			fileName: `${appName}.zip`,
-			initialPolicy: [
-				new PolicyStatement({
-					actions: ['dynamodb:BatchWriteItem'],
-					resources: [banditsTable.tableArn],
-				}),
-				new PolicyStatement({
-					actions: ['s3:*'],
-					resources: [
-						`arn:aws:s3:::gu-support-analytics/*`,
-						`arn:aws:s3:::gu-support-analytics`,
-					],
-				}),
-				new PolicyStatement({
-					effect: Effect.ALLOW,
-					actions: [
-						'athena:GetQueryExecution',
-						'athena:GetQueryResults',
-					],
-					resources: ['*'],
-				}),
-			],
-		});
-
-		const calculateTask = new LambdaInvoke(this, 'calculate-task', {
-			lambdaFunction: calculateLambda,
-			inputPath: '$.Payload',
-		});
 
 		const stateMachine = new StateMachine(this, 'state-machine', {
 			stateMachineName: `${appName}-${this.stage}`,
 			definitionBody: DefinitionBody.fromChainable(
 				getBanditTestsTask
-					.next(queryTask)
-					.next(
-						calculateTask.addRetry({
-							errors: ['QueryPendingError'],
-							interval: Duration.seconds(10),
-							maxAttempts: 5,
-						}),
-					)
+					.next(queryTask.addRetry({
```

Comment:
I don't think we need a retry anymore. Previously we used retries as a mechanism for polling the athena result.